<launch>
    <node pkg="webcam" type="webcam.py" name="webcam_node0" output="screen">
      <param name="topic_name" value="/camera0/compressed" />
      <param name="camera_id" type="int" value="0" />
      <param name="image_width" type= "int" value="1280" />
      <param name="image_height" type= "int" value="720" />
      <param name="frequency" type="int" value="10" />
    </node >
</launch>
